graphics.h

```diff
@@ -0,0 +1,30 @@
+#ifndef GRAPHICS_H
+#define GRAPHICS_H
+
+#include <SDL2/SDL.h>
+#include <iostream>
+
+
+ //constants
+ constexpr int kWindowWidth = 640;
+ constexpr int kWindowHeight = 480;
+ constexpr char kGameTitle[] = "DBZ";
+
+ //holds RGB values of background
```

Comment:
// Holds...

space between // and the first word of the comment. Capitalize the first word of the comment. Put a period at the end (or don't, but be consistent).